Processing/yOCTFindTissueSurface.m

```diff
@@ -2,19 +2,20 @@
 % This function processes OCT scan data to identify tissue surface from the
 % OCT image.
 % USAGE:
-%   surfacePosition = yOCTFindTissueSurface(meanAbs, dimensions [, outputUnits, isVisualize])
+%   [surfacePosition, x, y] = yOCTFindTissueSurface(logMeanAbs, dimensions, isVisualize)
 % INPUTS:
-%   logMeanAbs, dimensions - data structures generated by yOCTProcessScan.
+%   logMeanAbs, dimensions - data structures generated by yOCTProcessTiledScan.
 %       Converted to log scale
 %   isVisualize - set to true to generate image heatmap visualization
-%       figure. Default is false
+%       figure. Default is false.
 % OUTPUTS:
 %   - surfacePosition- 2D matrix. dimensions are (y,x). What
 %       height is image surface (measured from the top of the OCT scan).
-%       physical dimensions of surfacePosition are the same as dimensions.z.units
+%       Physical dimensions of surfacePosition are the same as dimensions.z.values
```

Comment:
Can you explain the sentence?

I recommend changing to:
% Physical **units** of surfacePosition are the same as dimensions.z.**units**